src/resolvers.ts

```diff
@@ -250,6 +251,14 @@ const resolvers = {
 
             convertErrorsToGqlErrors(deleteSubmissionResult)
             return deleteSubmissionResult.data
+        },
+
+        createAuditLog: async (_parent: unknown, args: { input: gqlType.CreateAuditLog })
```

Comment:
This actually should not be a mutation exposed to the front end. Instead it will just be logic that is triggered when any service mutation occurs.

Reply:
I also thought so but my only confusion was that how can I check the data that I put in the database against a schema? So I thought we should call the resolver?

Reply:
Do you mean in the test? 
You should be able to use a cypress intercept

Reply:
Ohhh I think you mean just saving to the db. We have the dbschema types for that. The whole reason why we have that is because there are some tables and fields in tables that should not be exposed to the public api. Creating Audit logs is a perfect example

Reply:
Ah I see, now the dbschema is more clear to me. ☺️I will rework this 🙏